pyproject.toml

```diff
@@ -168,6 +169,7 @@ pytest-cov = "^4.1.0"
 responses = "^0.23.3"
 mock = "^5.1.0"
 pytest-asyncio = "^0.21.1"
+ipykernel = "^6.29.4"
```

Comment:
Why is there a need to install `ipykernel` here?

Reply:
Not required